packages/jest-changed-files/src/index.ts

```diff
@@ -59,19 +57,8 @@ export const getChangedFilesForRoots = async (
 };
 
 export const findRepos = async (roots: Array<string>): Promise<Repos> => {
-  const gitRepos = await Promise.all(
-    roots.reduce<Array<RootPromise>>(
-      (promises, root) => promises.concat(findGitRoot(root)),
-      [],
-    ),
-  );
-  const hgRepos = await Promise.all(
-    roots.reduce<Array<RootPromise>>(
-      (promises, root) => promises.concat(findHgRoot(root)),
-      [],
-    ),
-  );
-
+  const gitRepos = await Promise.all(roots.map(findGitRoot));
```

Comment:
Could we await all of the promises at once?

Reply:
Yes, we can